f/dynamicViewDistance/f_setViewDistanceInit.sqf

```diff
@@ -0,0 +1,23 @@
+// FA3 - Dynamic View Distance
+// ====================================================================================
+
+if (!hasInterface) exitWith {};
+
+if (!isDedicated && (isNull player)) then
+{
+	waitUntil {sleep 0.1; !isNull player};
+};
+
+//Set view distance in case the player starts in vehicle
+[player, objNull, vehicle player] spawn f_fnc_ehSetViewDistance;
+
+// Add event handlers:
+// Note: Using PlayerViewChanged instead of GetInMan and GetOutMan to cover UAVs as well.
+f_ehIndex_dynamicViewDistance_0 = player addEventHandler ['SeatSwitchedMan',   f_fnc_ehSetViewDistance];
+f_ehIndex_dynamicViewDistance_1 = addMissionEventHandler ['PlayerViewChanged', f_fnc_ehSetViewDistance];
```

Comment:
Mildly concerned that PlayerViewChanged will get triggered while opening the map, looking through a scope, etc.

Reply:
There is nothing to worry about,
I tested this extensively, and it only seems to trigger when changing vehicles or when (the player starts or stops) operating a UAV.
The documentation seems to be correct in this case:
`Player view changes when player is changing body due to teamSwitch, gets in out of a vehicle or operates UAV for example. `